src/Files.App/App.xaml.cs

```diff
@@ -11,6 +11,7 @@
 using Windows.ApplicationModel.DataTransfer;
 using Windows.Storage;
 using Windows.UI.Notifications;
+using static Files.App.Helpers.Win32PInvoke;
```

Comment:
I don't like using static. If we focus on merge, you can ignore this.

Reply:
Removed the static using 👍